code/game/objects/structures/crates_lockers/closets/secure/medical.dm

```diff
@@ -57,8 +57,8 @@
 			new /obj/item/weapon/storage/backpack/medic(src)
 		else
 			new /obj/item/weapon/storage/backpack/satchel_med(src)
-		new /obj/item/clothing/under/rank/nursesuit (src)
-		new /obj/item/clothing/head/nursehat (src)

```

Comment:
Same with the comments here and on line 83.

Reply:
Same as the above answer